docs/source/physical_robot_installation_guide/index.rst

```diff
@@ -0,0 +1,109 @@
+===================================
```

Comment:
Somewhere early on you need to mention that Raspberry Pi is abbreviated as RPi. I know you had that before but it must have accidentally disappeared